frontend/src/common/components/naviconsidebar/NavButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFileLines, faHome } from "@fortawesome/free-solid-svg-icons";
+
+import "./NavButton.scss";
+import { getDefaultRequiredVal } from "../../helpers/util";
+
+export const NavButton = ({
+  type,
+  additionalClassNames,
+  onClick,
+}: {
+  type: "home" | "report";
```

Comment:
suggest creating a small enum here to reduce typos on matching `home` and `report`

Reply:
I'll create the enum equivalent as suggested.

Reply:
Enum equivalent created